doc/release-notes/0.18/0.18.md

```diff
@@ -0,0 +1,165 @@
+<!--
+* Copyright (c) 2019, 2019 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.18 release notes
+
+These release notes support the [Eclipse OpenJ9 0.18 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.18/plan).
```

Comment:
Just noticed this link is incorrect, it should be 0.18.0

Reply:
Seems it's always been wrong in previous release notes. Luckily the link takes you to https://projects.eclipse.org/projects/technology.openj9/releases where you can follow the links to find the plan.